.github/workflows/publish-pypi.yml

```diff
@@ -39,7 +38,7 @@ jobs:
       name: prod
       url: https://pypi.org/project/ultraplot/
     runs-on: ubuntu-latest
-    if: github.event_name == 'release' || (github.event_name == 'push' && needs.publish-pypi-test.result == 'success')
```

Comment:
Not sure if we  can remove that last part, but otherwise LGTM.

Reply:
We can since the `needs` block will only run the second job if the first suceeds.